src/main/java/it/eng/idsa/dataapp/util/MessageUtil.java

```diff
@@ -77,4 +162,39 @@ private String createContractAgreement(Path dataLakeDirectory) {
 		}
 		return contractAgreement;
 	}
+	
+	private String getSelfDescription() {
```

Comment:
Can this one return BaseConnector (or even Connector) class instead String?

Reply:
Sure